src/backend/weaviate/weaviate_manager.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import weaviate
+from weaviate import Client
+
+#config logging
+logging.basicConfig(level=logging.DEBUG,format='%(asctime)s - %(levelname)s - %(message)s')
+logger = logging.getLogger(__name__)
+
+class WeaviateManager:
+    def __init__(self):
+
+        try:
+            self.client=weaviate.Client("http://weaviate:8080")
```

Comment:
Can this value "http://weaviate:8080" be obtained from Environment values set in docker compose file instead of hard coding it